meta-ostro-xt/recipes-swupd/swupd-client/swupd-client_%.bbappend

```diff
@@ -1,10 +1,17 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI_append = " \
+    file://efi_combo_updater.sh \
```

Comment:
do_install fails with
```
07:55:06 | install: cannot stat '/srv/jenkins/workspace/builder-slot-0/build/tmp-glibc/work/corei7-64-ostro-linux/swupd-client/3.6.0+gitAUTOINC+f4000c5b22-r0/efi-combo-trigger.service.new': No such file or directory
```

Not sure why fetcher works correctly but looks like ```file://efi-combo-trigger.service.new``` is missing